### PR TITLE
play23 -- test passed!

### PR DESCRIPTION
**Pull request by raymondng**
Dependencies updated

@andrewconner

### CODE REVIEW

**Comment by andrewconner** [comment]
Yep

File: kifi-backend/modules/common/conf/dev-akka-sync.conf [hunk 1]
```diff
@@ -9,6 +9,7 @@ play {
     loggers = ["akka.event.slf4j.Slf4jLogger"]
     loglevel = DEBUG
     log-config-on-start = false
+    jvm-exit-on-fatal-error = off
```
Comment (eishay) [inline kifi-backend/modules/common/conf/dev-akka-sync.conf:12]: i thought we already have it

File: kifi-backend/project/Global.scala [hunk 2]
```diff
@@ -50,11 +50,11 @@ object Global {
     jdbc, // todo(andrew): move to sqldb when we discover a way to get Play to support multiple play.plugins files.
     cache,
     ws,
-    "com.typesafe.play.plugins" %% "play-statsd" % "2.2.0" exclude("play", "*"),
-    "com.typesafe" %% "play-plugins-mailer" % "2.2.0" exclude("play", "*"),
+    "com.typesafe.akka" %% "akka-testkit" % "2.3.4" exclude("play", "*"),
```
Comment (eishay) [inline kifi-backend/project/Global.scala:53]: great!

File: kifi-backend/modules/common/conf/dev-akka-sync.conf (hunk 1 above)
Comment (raymondng) [inline kifi-backend/modules/common/conf/dev-akka-sync.conf:12]: we have it in prod but not dev. dev was bombing out due to akka upgrade breakage, so i added this to debug. turns out we had some incompatible libraries -- see `Global.scala` -- fixed.

**Pull request merged**